pkgs/development/python-modules/requests-gssapi/default.nix

```
@@ -0,0 +1,27 @@
+{ fetchPypi
+, lib
+, python3Packages
+}:
+python3Packages.buildPythonPackage rec {
+  pname = "requests-gssapi";
+  version = "1.2.3";
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-IHhFCJgUAfcVPJM+7QlTOJM6QIGNplolnb8tgNzLFQ4=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    gssapi
+    requests
+  ];
+  pythonImportCheck = [ "requests_gssapi" ];
+
+  meta = with lib; {
+    description = "A GSSAPI authentication handler for python-requests";
+    homepage = "https://github.com/pythongssapi/requests-gssapi";
+    maintainers = with maintainers; [ javimerino ];
```

Tools (e.g., `nix-update`) can add a link to the changelog if the changelog entry is present.

```suggestion
    homepage = "https://github.com/pythongssapi/requests-gssapi";
    changelog = "https://github.com/pythongssapi/requests-gssapi/blob/v${version}/HISTORY.rst";
    maintainers = with maintainers; [ javimerino ];
```

---

Today I learned! Thanks